guacamole/src/main/frontend/src/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Guacamole",
+  "short_name": "Guacamole",
+  "description": "Guacamole",
+  "start_url": ".",
+  "display": "standalone",
+  "icons": [
+    {
+      "src": "./images/logo-vector.svg",
+      "type": "image/svg+xml",
+      "sizes": "512x512"
+    },
+    {
+      "src": "./images/logo-512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    },
+    {
+      "src": "./images/logo-192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    }
+  ]
+}
```

Comment:
Any reason this is spaces with 2 character indentations, instead of the 4 characters that is common across (most of) the rest of the Guacamole source code?

Reply:
Bad configuration of my IDE, I corrected it!